web/assets/scss/pro-hub.scss

```diff
@@ -49,4 +49,24 @@
   .govuk-form-group {
     max-width: 40em;
   }
+
+  span.rag {
+    font-weight: bold;
+    &.green {
+      color: #28a197 !important;
```

Comment:
It's generally adviseable to avoid `!important` as it can cause confusion. It shouldn't be necessary here either, as this style sheet is applied last so has greater specificity.